Directory.Build.props

```diff
@@ -1,5 +1,8 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<ItemGroup>
-		<PackageReference Include="Messerli.CodeStyle" Version="1.0.0" PrivateAssets="all" />
+		<PackageReference Include="Messerli.CodeStyle" PrivateAssets="all" />
 	</ItemGroup>
+	<PropertyGroup>
+		<DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
```

Comment:
remove this

Reply:
https://github.com/messerli-informatik-ag/file-system/pull/9/commits/f90d668d2b15f7e98146beaee36a11e5ca3fb8a9